samples/CMakeLists.txt

```diff
@@ -2,8 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-# Samples do not need to be built for NPM package
-if(CPACK_GENERATOR STREQUAL "NPM")
+if(NOT ENABLE_SAMPLES)
     return()
```

Comment:
it's not required as handled in main CMakeLists.txt

Reply:
Right, haven't seen